Automattic/jetpack
# Add why i need backup section to connect page

## Proposed changes:

* Adds new backup connection component for Why I need VaultPress Backup
* Uses new section on bottom of connect page

### Other information:


## Jetpack product discussion

P2: p1HpG7-kd7-p2#comment-60237
Figma: bSsQjyTPK8VIfopwI4fdIU-fi-1_29

## Does this pull request change what data or activity we track or use?

No

## Testing instructions:

1. Checkout this branch locally or with the Jetpack Beta plugin
2. DONT connect Jetpack to your wordpress account
3. Install the Jetpack VaultPress Backup plugin
4. If you are not redirected there, go to `/wp-admin/admin.php?page=jetpack-backup`
5. See the new Why I need VP Backup section and make sure it matches the comps/looks good on mobile
![image](https://github.com/Automattic/jetpack/assets/65001528/1bfe2af4-c082-415d-80a1-bd791fb65920)
